src/Interface.jl

```diff
@@ -243,3 +243,33 @@ struct Outlet{V} <: AbstractBoundaryInterface
     F::Function
 end
 export Outlet
+
+
+struct Condensation{S,V<:AbstractArray,U<:Real,X<:Real} <: AbstractBoundaryInterface
```

Comment:
Yes this is the one-way condensation because the evaporation direction depends on the liquid composition. I'm thinking to change the name `condensation` to `VaportoLiquidMassTransfer`, and `evaporation` to `LiquidtoVaporMassTransfer`. What do you think?

Reply:
Evaporation and condensation are now combined to one interface